docs/src/solvers/ode_solve.md

```diff
@@ -498,6 +498,10 @@ These methods also have option `nlsolve` same as SDIRK methods. These methods al
   stiff-aware 3rd order interpolant. 4th order on linear parabolic problems and
   3rd order accurate on nonlinear parabolic problems (as opposed to lower if not
   corrected).
+- `Rodas4P2` - A 4th order A-stable stiffly stable Rosenbrock method with a
+  stiff-aware 3rd order interpolant. 4th order on linear parabolic problems and
+  3rd order accurate on nonlinear parabolic problems. It is an improvement of Roadas4P 
+  and in case of inexact Jacobians a second order W method.
```

Comment:
oh cool, didn't realize it was also a W method. This should get some good benchmarks